package.json

```diff
@@ -29,10 +29,10 @@
     "popper.js": "1.14.3",
     "prop-types": "^15.5.10",
     "query-string": "^5.1.1",
-    "react": "^16.4.0",
+    "react": "^16.4.1",
```

Comment:
Any reason to not also go to 16.4.2 here too?

Reply:
why not